cfgov/v1/atomic_elements/organisms.py

```diff
@@ -6,6 +6,13 @@
 from ..util import ref
 from ..models.snippets import Contact as ContactSnippetClass
 
+JS_ORGANISMS = [
```

Comment:
Do you still want to merge this PR, or is it better to wait for the future, improved one?